nixos/modules/services/monitoring/netdata.nix

```diff
@@ -31,9 +46,9 @@ in {
       };
 
       configText = mkOption {
-        type = types.lines;
-        default = "";
-        description = "netdata.conf configuration.";
+        type = types.nullOr types.str;
```

Comment:
any reason not to keep using `types.lines` here?

Reply:
Oh, no reason. I didn't realize this got flipped. I'll switch it back.